pkg/config/sysprep.go

```diff
@@ -67,31 +67,38 @@ func validateAutounattendPresence(dirPath string) error {
 func CreateSysprepDisks(vmi *v1.VirtualMachineInstance) error {
 	for _, volume := range vmi.Spec.Volumes {
 		if volume.Sysprep != nil {
-			if sysprepVolumeHasContents(*volume.Sysprep) {
-				var filesPath []string
+			err := createSysprepDisk(volume)
```

Comment:
Could you please also add a description to the commit message saying what the commit does